src/Backoffice/City/App/Requests/StoreCityRequest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\City\App\Requests;
+
+class StoreCityRequest extends BaseCityRequest
+{
+}
```

Comment:
if both requests are the exact same, then i would just do the one upsert request. the extends one is for when you have 2 that are almost the same, but change, for example, in the toDto, if they are the same, then we can just do `UpsertCityRequest`